src/main/js/isd.js

```diff
@@ -59,22 +59,90 @@
 
         };
 
-        /* process regions */
+        /* Filter body contents - Only process what we need within the offset and discard regions not applicable to the content */
+        var body = {};
+        var activeRegions = {};
+
+        /* gather any regions that might have showBackground="always" and show a background */
+        initialShowBackground = tt.head.styling.initials[imscStyles.byName.showBackground.qname];
+        initialbackgroundColor = tt.head.styling.initials[imscStyles.byName.backgroundColor.qname];
+        initialbackgroundImage = tt.head.styling.initials[imscStyles.byName.backgroundImage.qname];
+        for (var layout_child in tt.head.layout.regions)
+        {
+            if (tt.head.layout.regions.hasOwnProperty(layout_child)) {
+                region = tt.head.layout.regions[layout_child];
+                showBackground = region.styleAttrs[imscStyles.byName.showBackground.qname] || initialShowBackground;
+                backgroundColor = region.styleAttrs[imscStyles.byName.backgroundColor.qname] || initialbackgroundColor;
+                backgroundImage = region.styleAttrs[imscStyles.byName.backgroundImage.qname] || initialbackgroundImage;
+                activeRegions[region.id] = (
+                    (showBackground === 'always' || showBackground === undefined) &&
+                    (backgroundColor !== undefined || backgroundImage !== undefined) &&
```

Comment:
```suggestion
```

Reply:
`backgroundImage` check removed in 9c3dc7b but `backgroundColor` check retained.